### PR TITLE
[DO NOT MERGE] feat: remove Matrix bis

### PR DESCRIPTION
**Pull request by Tbaut**
To be merged with https://github.com/novasamatech/nova-spektr/pull/1800

### CODE REVIEW

**Comment by Tbaut** [comment]
This is a revert of f23851c57e33f9319888c669fcc19cda97ada470
Yet I see it contains unrelated changes, so I'll change what I see, or comment, see below, but please point out @tuul-wq anything that seems outside of the scope or the matrix removal and should eventually be kept as is.

File: src/renderer/features/notifications/NotificationsList/ui/notifies/MultisigCreatedNotification.tsx [hunk 1]
```diff
@@ -5,13 +5,14 @@ import type { MultisigCreated } from '@shared/core';
...
 import { useI18n } from '@app/providers';
+import { ChainTitle } from '@entities/chain';
 
 type Props = {
   notification: MultisigCreated;
 };
 
 export const MultisigCreatedNotification = ({
-  notification: { threshold, signatories, multisigAccountName },
+  notification: { threshold, signatories, multisigAccountName, chainId },
```
Comment (Tbaut) [inline src/renderer/features/notifications/NotificationsList/ui/notifies/MultisigCreatedNotification.tsx:15]: Not sure about this change for instance, it doesn't seem to have anything to do with matrix

File: src/renderer/shared/api/xcm/service/xcmService.ts [hunk 2]
```diff
@@ -196,9 +196,9 @@ type XTokensPayload = ParsedPayload & {
 function parseXcmPalletExtrinsic(args: Omit<XcmPalletTransferArgs, 'feeAssetItem' | 'weightLimit'>): XcmPalletPayload {
   const xcmVersion = Object.keys(args.dest as Object)[0];
 
-  const assetInterior = get(args.assets, `${xcmVersion}[0].id.Concrete.interior`) as unknown as Object;
-  const destInterior = get(args.dest, `${xcmVersion}.interior`) as unknown as Object;
-  const beneficiaryInterior = get(args.beneficiary, `${xcmVersion}.interior`) as unknown as Object;
+  const assetInterior = get(args.assets, `${xcmVersion}[0].id.Concrete.interior`) as Object;
```
Comment (Tbaut) [inline src/renderer/shared/api/xcm/service/xcmService.ts]: idem in this file, which version do we want to keep?

File: src/renderer/shared/ui/Dropdowns/DropdownButton/DropdownButton.tsx [hunk 3]
```diff
@@ -17,62 +17,58 @@ type Props = {
   options: ButtonDropdownOption[];
 } & Omit<ButtonProps, 'children' | 'suffixElement' | 'onClick'>;
 
-export const DropdownButton = ({ options, title, disabled, className, ...buttonProps }: Props) => {
-  const { ref, ...restProps } = buttonProps;
+export const DropdownButton = ({ options, title, disabled, className, ...buttonProps }: Props) => (
```
Comment (Tbaut) [inline src/renderer/shared/ui/Dropdowns/DropdownButton/DropdownButton.tsx:20]: And this file.

**Review by tuul-wq — APPROVED** [review]

**Review by pgolovkin — APPROVED** [review]

**Comment by pgolovkin** [comment]
@Tbaut nice PR! Thanks for removing Matrix.
I'd like to confirm that this PR will be merged when new flow of the Multisig creation will be implemented, right?

**Comment by Tbaut** [comment]
Yes, it's to be merged together with #1800 as said in the description

**Comment by Tbaut** [comment]
closing in favor of https://github.com/novasamatech/nova-spektr/pull/2380

**Pull request closed without merging**